spec/features/login_page_spec.rb

```diff
@@ -0,0 +1,98 @@
+describe 'login_page' do
+  before { visit '/login' }
+
+  context 'Visibility' do
+    it 'has Email field' do
+      expect(page).to have_css('input#spree_user_email')
+    end
+
+    it 'has Password field' do
+      expect(page).to have_css('input#spree_user_password')
+    end
+
+    it 'has Remember_me checkbox' do
+      expect(page).to have_css('input#spree_user_remember_me')
+      expect(page).to have_css('label', text: 'Remember me')
+    end
+  end
+
+  context 'Login process' do
+    it 'can Log in as a User' do
+      fill_in 'spree_user_email', with: ENV['USERNAME_SPREE']
+      fill_in 'spree_user_password', with: ENV['PASSWORD_SPREE']
+      find_button('Login').click
+      expect(page).to have_css('a', text: 'Logout')
+      expect(find('.alert-success').text).to eql 'Logged in successfully'
```

Comment:
I'd suggest moving it to a separate variable, e.g `alert_text = find('.alert-success').text => expect(alert_text).to eq ...`
This makes it more readable. Same goes for other specs
Also, why are you using `eql` instead of `eq`? This doesn't make a lot of sense in case of strings 🙂 
https://stackoverflow.com/questions/32926817/rspec-eq-vs-eql-in-expect-tests